solarforecastarbiter/datamodel.py

```diff
@@ -419,6 +419,130 @@ class Observation(BaseModel):
     __post_init__ = __set_units__
 
 
+@dataclass(frozen=True)
+class AggregateObservation(BaseModel):
+    """
+    Class for keeping track of Observations in Aggregates
+    and when they are added and removed from the Aggregate
+
+    Parameters
+    ----------
+    observation : Observation
+        The Observation object that is part of the Aggregate
+    effective_from : pandas.Timestamp
+        The effective datetime of when the Observation should be
```

Comment:
```suggestion
        The datetime of when the Observation should be
```

Reply:
The use of effective in the kwarg name is clear to me, but the use in the description makes me think "effective" as in "effectively".